load_sample_data.sh

```diff
@@ -17,31 +17,40 @@ docker-compose exec -T "$WEB_SERVICE_NAME" /bin/bash -c "python3 manage.py shell
 
 # Migrate analytics
 docker-compose exec -T "$ANALYTICS_SERVICE_NAME" /bin/bash -c "PYTHONPATH=. pipenv run alembic upgrade head"
-# Copy table `image` from database to upstream database
-docker-compose exec -T "$UPSTREAM_DB_SERVICE_NAME" /bin/bash -c "PGPASSWORD=deploy pg_dump -s -t image -U deploy -d openledger -h db | psql -U deploy -d openledger"
+
 # Load content providers
 docker-compose exec -T "$DB_SERVICE_NAME" /bin/bash -c "psql -U deploy -d openledger <<-EOF
 	INSERT INTO content_provider (created_on, provider_identifier, provider_name, domain_name, filter_content, media_type) VALUES (now(), 'flickr', 'Flickr', 'https://www.flickr.com', false, 'image'), (now(), 'behance', 'Behance', 'https://www.behance.net', false, 'image');
 	EOF"
-# Load sample data, including content providers
+
+docker-compose exec -T "$UPSTREAM_DB_SERVICE_NAME" /bin/bash -c "psql -U deploy -d openledger <<-EOF
+	CREATE TABLE content_provider(provider_identifier varchar(50), provider_name varchar(250), created_on timestamp, domain_name varchar(500), filter_content boolean, notes text, media_type varchar(80));
+	INSERT INTO content_provider (created_on, provider_identifier, provider_name, domain_name, filter_content, media_type) VALUES (now(), 'flickr', 'Flickr', 'https://www.flickr.com', false, 'image'), (now(), 'behance', 'Behance', 'https://www.behance.net', false, 'image');
+	EOF"
+
+# Load sample data for images
+docker-compose exec -T "$UPSTREAM_DB_SERVICE_NAME" /bin/bash -c "PGPASSWORD=deploy pg_dump -s -t image -U deploy -d openledger -h db | psql -U deploy -d openledger"
 docker-compose exec -T "$UPSTREAM_DB_SERVICE_NAME" /bin/bash -c "psql -U deploy -d openledger <<-EOF
 	ALTER TABLE image RENAME TO image_view;
 	ALTER TABLE image_view ADD COLUMN standardized_popularity double precision;
-	CREATE TABLE content_provider(provider_identifier varchar(50), provider_name varchar(250), created_on timestamp, domain_name varchar(500), filter_content boolean, notes text, media_type varchar(80));
-	INSERT INTO content_provider (created_on, provider_identifier, provider_name, domain_name, filter_content) VALUES (now(), 'flickr', 'Flickr', 'https://www.flickr.com', false, 'image'), (now(), 'behance', 'Behance', 'https://www.behance.net', false, 'image');
 	\copy image_view (id,created_on,updated_on,identifier,provider,source,foreign_identifier,foreign_landing_url,url,thumbnail,width,height,filesize,license,license_version,creator,creator_url,title,tags_list,last_synced_with_source,removed_from_source,meta_data,tags,watermarked,view_count,standardized_popularity) from './sample_data/sample_data.csv' with (FORMAT csv, HEADER true)
```

Comment:
```suggestion
	\copy image_view (identifier,created_on,updated_on,ingestion_type,provider,source,foreign_identifier,foreign_landing_url,url,thumbnail,width,height,filesize,license,license_version,creator,creator_url,title,meta_data,tags,watermarked,last_synced_with_source,removed_from_source,standardized_popularity) from './sample_data/sample_data.csv' with (FORMAT csv, HEADER true)
```

Reply:
Will just removing the field names from the `\copy` command work or do we have to delete the columns from the CSV as well?

Reply:
Updated in 75ca546.